app/uk/gov/hmrc/customsservicestatusfrontend/controllers/test/TestController.scala

```diff
@@ -46,14 +46,14 @@ class TestController @Inject() (
     }
 
   val showAvailable: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(dashboardPage(AVAILABLE, Instant.now())))
+    Future.successful(Ok(dashboardPage(AVAILABLE, Instant.now(), "GVMS haulier")))
```

Comment:
Each of these 3 calls in this file needs to change to now be just the service id "haulier"

Reply:
updated.